config/initializers/date_formats.rb

```diff
@@ -13,6 +13,8 @@
              '%e %B %Y at %l%P (midday)'
            elsif time >= time.midnight && time <= time.midnight.end_of_minute
              '%e %B %Y at %l%P (midnight)'
+           elsif time.min.zero?
+             '%e %B %Y at %l%P'
```

Comment:
@adamsilver this updates the time displays everywhere in the application (Apply & Manage). So anything that said `1st January 2021 at 11:00am` will now say `1st January 2021 at 11am` and `1:00pm` will change to just `1pm`. Does that sound OK?